### ISSUE TITLE
Calculate percentage for weighted and unweighted variable

### ISSUE THREAD

**Issue by borsna** [MetadataTransform/ddi-xslt#24]
Bug in `transformations/fo/from-ddi-2.5`
result:
[export2.pdf](https://github.com/MetadataTransform/ddi-xslt/files/13599035/export2.pdf)

**Comment by borsna** [comment]
Will take a look in a few days and get back to you (hopefully with a nice fix 🙂)

**Comment by lubitchv** [comment]
My colleague @JayanthyChengan found that Percentage category does not shows up because xsl tries to build bars in a cell for Percentages. By removing the bars and leaving $catgry-pct the percentage shows up now. I updated the repository with this fix.

**Comment by borsna** [comment]
great fix @lubitchv! 
Did this resolve all the cases for percentage for weighted and unweighted variable now showing up?

**Comment by lubitchv** [comment]
Yes, if variable is weighted then it shows weighted percentage and it if is unweighted then it shows unweighted percentage. I opened a pull request to IQSS Harvard to incorporate it in Dataverse. https://github.com/IQSS/dataverse/pull/10295

**Comment by borsna** [comment]
Great work @lubitchv & @JayanthyChengan!
Could you show the integration in DataVerse at the next virtual DDI Developers meeting?
No date set yet but we will certainly keep it at 16:00 CET.

**Comment by lubitchv** [comment]
Yes, of course.
